cognite/powerops/custom_modules/power_model_v1/data_models/containers/Case.container.yaml

```diff
@@ -10,19 +10,51 @@ properties:
     autoIncrement: false
     name: scenario
     description: The scenario to use when running this case
+  case:
+    type:
+      list: false
+      type: file
+    nullable: true
+    autoIncrement: false
+    name: case
+    description: The resulting case file to send to shop after preprosessor has performed mappings on model template
+  reservoirMapping:
+    type:
+      list: false # Only one per case?
+      type: file
+    nullable: true
+    autoIncrement: false
+    name: reservoirMapping
+    description: Cut file reservoir mapping
+  moduleSeries:
+    type:
+      list: true # may be multiple per case
+      type: file
+    nullable: true
+    autoIncrement: false
+    name: moduleSeries
+    description: Module series file(s)
+  cutFiles:
+    type:
+      list: true # may be multiple per case
+      type: file
+    nullable: true
+    autoIncrement: false
+    name: cutFiles
+    description: Cut file(s)
```

Comment:
Not sure if we are covering the relevant/right set of files - it could be an option (like you suggested before) to just have a generic list of files, but then we may have to add a way to define the order they should be read in?